tolerance-review/AppFlow.adoc

```diff
@@ -0,0 +1,18 @@
+:gls_prefix:
+
+// Do not modify section headers, as they are used by flamel and the translation process.
+= pass:a,n[{gls_res_outcomes}]
+
+You should be able to apply different fault tolerance policies to Quarkus applications given identified potential faults.
+
+== Present App Flow
+
+The present lab makes use of Session, Speaker and Schedule services.
+
+* `Aliveness` and `Readiness` checks are implemented.
+* Prepare the Session service to use `fault tolerance` features.
+* Create a `fallback` and enable the fault tolerance policy.
+* Create a fallback method for returning `non-enriched` speaker information.
```

Comment:
These two seem duplicated, am I right?

Reply:
There is a very slight difference between the two. In the first, we are applying _Fallback policies to an existing method_ whereas in the other _we are creating a fallback method._

Reply:
Afaik, applying a Fallback policy requires an existing fallback method, so both steps should be combined into a single one IMO

Reply:
Okay. I will merge both the steps.